corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -427,4 +427,7 @@
     {% block js-inline %}{% endblock js-inline %}
 
   </body>
+  {% if show_status_page %}
+    <script src="https://p3s9fvl6gvhr.statuspage.io/embed/script.js"></script>
```

Comment:
was there a reason to include this script tag outside of the `<body>`? this is generally not considered valid HTML.

For these kinds of scripts, it is best to include it before the `{% block js %}` tags in `hqwebapp/base.html`. It would be good to see how `show_prepaid_modal` does it. I feel like the way this is included may interfere with `requirejs`, but let's at least try and see.

Reply:
I think this was recommended by the website that provided the scripts, but I'm not opposed to changing it if it's not a good idea to have this outside of body